### PR TITLE
add hint for "universal" mode only

### PR DESCRIPTION
**Pull request by manniL**

### CODE REVIEW

**Comment by manniL** [comment]
Depends on #11 for CI fix

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@
 
 ✓ ...and more! (see [http-proxy-middleware](https://github.com/chimurai/http-proxy-middleware) docs)
 
+⚠ Only working when using Nuxt in `ssr`/`universal` mode
```
Comment (pi0) [inline README.md]: Actually, this works with SPA mode as well when running `nuxt start --spa` (or using config). I think conditions should be changed to check for generating and static deployment of nuxt. For the second one, we can't detect it as this is up to the deployment method. I suggest just warning users here about this fact.

Comment (manniL) [reply README.md]: Dang, you are right!

So only a warning in the `generate:before` hook?

Comment (pi0) [reply README.md]: Seems good 👍

**Pull request merged**